### ISSUE TITLE
Change avatar of any user as administrator

### ISSUE THREAD

**Issue by helgator** [Chocobozzz/PeerTube#6618]
### Describe the problem to be solved

How do I change the avatar of a user as the administrator?

### Describe the solution you would like

As the administrator I can change the avatar of a user.

**Comment by JohnXLivingston** [comment]
What is the point of this feature?
If think that admins should not be able to modify users personal data* (just to be clear: i'm not a Peertube maintainer, i just give my personal thoughts on this subject).

If it is because of an inappropriate avatar, user should just be ask to change it. And if they refuse, they should be banned or muted.


*: except e-mail address, to help users recover their accounts if their mail account does not exist anymore, or if there was a typo.

**Comment by helgator** [comment]
My users (teachers) are either technically not able to upload their avatar or just dont care anymore once they uploaded their content. I as the moderator have to do everything for them: select the correct category, change channel, set correct language, upload avatar, create appropriate video thumbnail, add license, ...

**Comment by stevespaw** [comment]
I agree 100% - I would love an "impersonation" feature as an administrator. Just Like NextCloud and many others, It could enabled or disabled. Probably best as a plug-in, but I have no idea if that would work.

**Comment by JohnXLivingston** [comment]
> > If it is because of an inappropriate avatar, user should just be ask to change it. And if they refuse, they should be banned or muted.
> 
> Their avatar will still publicly available even after a ban. People who came from search engine, direct links, or even from comments and videos, can still see those avatars.

Hum, are you sure? I think last time i checked, banned users profiles were not accessible (unless you are connected with a moderation/admin account).

**Comment by helgator** [comment]
> > > If it is because of an inappropriate avatar, user should just be ask to change it. And if they refuse, they should be banned or muted.
> > 
> > 
> > Their avatar will still publicly available even after a ban. People who came from search engine, direct links, or even from comments and videos, can still see those avatars.
> 
> Hum, are you sure? I think last time i checked, banned users profiles were not accessible (unless you are connected with a moderation/admin account).

This is not the point at all. I do **not** want to ban anyone. I as the admin just want to help my technically and timely overwhelmed users to maintain the instance.

**Comment by JohnXLivingston** [comment]
> This is not the point at all. I do **not** want to ban anyone. I as the admin just want to help my technically and timely overwhelmed users to maintain the instance.

Ok, but can we discuss the requested feature more widely? I was not answering your comment. You are not the only Peertube user.

**Comment by JohnXLivingston** [comment]
> I am pretty sure. I even tried this a moment ago just to make sure of that.
> I can see profile picture in comment section and when I click on it, I got to see the user's profile.I tried this in Incognito tab without login.

Ok, you are right. Might be a regression. I opened a separate issue for that:
https://github.com/Chocobozzz/PeerTube/issues/6675
